test/lit/passes/local-subtyping.wast

```diff
@@ -202,4 +202,25 @@
       )
     )
   )
+
+  ;; CHECK:      (func $nondefaultable
+  ;; CHECK-NEXT:  (local $x (anyref anyref))
+  ;; CHECK-NEXT:  (local.set $x
+  ;; CHECK-NEXT:   (tuple.make
+  ;; CHECK-NEXT:    (ref.func $i32)
+  ;; CHECK-NEXT:    (ref.func $i32)
+  ;; CHECK-NEXT:   )
+  ;; CHECK-NEXT:  )
+  ;; CHECK-NEXT: )
+  (func $nondefaultable
+    (local $x (anyref anyref))
```

Comment:
It's a little unfortunate that we can't at least make this `(funcref funcref)`, which is more refined than `(anyref anyref)` but is still defaultable. Also, it's too bad that the non-nullable local support doesn't handle tuples, but I guess our tuple lowering still needs to be better for that to make any sense.